multy_core/src/ethereum/ethereum_transaction_builder_multisig.cpp

```diff
@@ -0,0 +1,246 @@
+/* Copyright 2018 by Multy.io
+ * Licensed under Multy.io license.
+ *
+ * See LICENSE for details
+ */
+
+#include "multy_core/src/ethereum/ethereum_transaction_builder_multisig.h"
+
+#include "multy_core/error.h"
+
+#include "multy_core/src/ethereum/ethereum_account.h"
+#include "multy_core/src/ethereum/ethereum_address.h"
+
+#include "multy_core/src/api/big_int_impl.h"
+#include "multy_core/src/api/properties_impl.h"
+#include "multy_core/src/api/transaction_builder_impl.h"
+#include "multy_core/src/enum_name_map.h"
+#include "multy_core/src/exception.h"
+#include "multy_core/src/exception_stream.h"
+#include "multy_core/src/property_predicates.h"
+#include "multy_core/src/utility.h"
+
+#include <string>
+#include <regex>
+#include <unordered_map>
+
+namespace
+{
+using namespace multy_core::internal;
+
+class MultisigTransactionBuilderBase : public TransactionBuilder
+{
+protected:
+    MultisigTransactionBuilderBase(const std::string& name)
+        : m_properties(ERROR_SCOPE_TRANSACTION_BUILDER, name)
+    {}
+
+    Properties& get_properties() override
+    {
+        return m_properties;
+    }
+
+    ~MultisigTransactionBuilderBase()
+    {}
+
+    void validate(const CodeLocation& location) const
+    {
+        m_properties.validate(location);
+    }
+
+protected:
+    Properties m_properties;
```

Comment:
Add variable transaction and generate it in constructor.

Reply:
Nope, since transactions are not clonable and builder must return new copy of tx for each call of `TransactionBuilder::make_transaction()`. Right now we are saving `const Account&` in order to make transaction later.